### ISSUE TITLE
Enhancement, components styles on it's own directory

### ISSUE THREAD

**Issue by joeyhipolito** [johnpapa/generator-hottowel#94]
I might be wrong, but I think it would be better if components's styles reside on it's own folder, for example the styles for the ht-sidebar directive should be on the same folder, ht-sidebar.

**Comment by rllola** [comment]
+1
I also think it is  a good idea. I find it convenient to have all the files in the same directory when working on a feature.

**Comment by johnpapa** [comment]
yes they should normally. feel free to make a pr.

but, it could be argued that layout is part of the core. but i tend to lean where you are.

**Comment by Codermar** [comment]
I started to work with hottowel with the intent of moving my project to its structure which I think it's great. My inclination is to promote the ability to theme the apps and I have in mind the idea of combining ideas of the hottowel seed structure with the one I played with http://themicon.co/theme/angle/v2.5/backend-angular

Some ideas in that seed are pretty good and in general the code is organized fairly well giving you the ability to  change layouts and themes easily. Anyway, I am just getting started. I'll keep you updated but one thing I agree with is separating the theming with the rest of the app. so themes can be worked on and maintained outside the general app.

I might be interested in contributing if my experiment goes well :-)
